a10_octavia/controller/worker/controller_worker.py

```diff
@@ -472,13 +475,13 @@ def create_member(self, member_id):
         :raises NoSuitablePool: Unable to find the node pool
         """
 
+        time.sleep(60)
```

Comment:
Don't use sleep(60) but use a general API to check/re-check and wait instead.

Reply:
Fixed